hack/make/.binary

```diff
@@ -18,7 +18,8 @@ source "${MAKEDIR}/.go-autogen"
 
 	case "$(go env GOOS)/$(go env GOARCH)" in
 		linux/arm)
-			case "${GOARM}" in
+			# specify name of the target ARM architecture for cross-compilation
+			case "$(go env GOARM)" in
```

Comment:
Please squash this into the previous commit as it was an oversight introduced in the previous commit.